src/data/equiv/mul_add.lean

```diff
@@ -418,6 +418,19 @@ lemma Pi_congr_right_trans {η : Type*}
   (es : ∀ j, Ms j ≃* Ns j) (fs : ∀ j, Ns j ≃* Ps j) :
   (Pi_congr_right es).trans (Pi_congr_right fs) = (Pi_congr_right $ λ i, (es i).trans (fs i)) := rfl
 
+
+/-- A family indexed by a unique type is equivalent to the element at the default value -/
+@[to_additive add_equiv.Pi_unique "A family indexed by a unique type is equivalent to the
+element at the default value"]
+def Pi_unique
+  {η : Type*} (M : η → Type*) [Π (j : η), mul_one_class (M j)] [unique η] :
+  (Π (j : η), M j) ≃* M default :=
+{ to_fun := λ f, f default,
+  inv_fun := λ x, λ j, by { rw unique.eq_default j, exact x, },
+  left_inv := λ f, by { ext j, rw unique.eq_default j, reflexivity, },
+  right_inv := λ x, rfl,
```

Comment:
Does this exist already for types with no multiplicative structure in the `equiv` namespace?

Reply:
I dont know. i guess ill check and add if needed

Reply:
Added `Pi_subsingleton` and `Pi_unique` to `equiv` as well.